modules/core/src/main/java/org/apache/ignite/cluster/ClusterMetrics.java

```diff
@@ -347,15 +347,15 @@
      * Gets percentage of time this node is busy executing jobs vs. idling.
      *
      * @return Percentage of time this node is busy (value is less than
-     *      or equal to {@code 1} and greater than or equal to {@code 0})
+     *      or equal to {@code 100} and greater than or equal to {@code 0})
```

Comment:
You change the public contract with these changes. Let's leave this as is.

Reply:
It is not logical. Method name percentage of use, but to output the share

Reply:
I agree, but some users can already have monitoring system configured for value range [0;1], and we can't just break this in minor fix which even doesn't touch this directly. In such cases we need to continue living with it - and create a ticket to fix this in Ignite 3.

Reply:
Nobody used the metrics, and when they tried they noticed an error, in fact, this is why a ticket was opened